src/components/Network.js

```diff
@@ -164,7 +164,7 @@ class Network extends React.Component {
         const { synced_percent, general_percent } = this.getSyncProgressPercent(sync_data);
         const sync_state_description = SyncStatesDescription[sync_data.state] || SyncStatesDescription[SyncStates.UNKNOWN];
 
-        const entrypoints = peer.entrypoints || [];
+        const entrypoints = conn.entrypoints || [];
```

Comment:
Is it a bug fix?

Reply:
it's from another PR that it's here by accident. Removed.